api/spec/src/app/stripe.tsp

```diff
@@ -0,0 +1,345 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi3";
+
+import "./app.tsp";
+import "./capability.tsp";
+import "../errors.tsp";
+import "../oauth.tsp";
+import "../pagination.tsp";
+import "../types.tsp";
+
+using OpenMeter.App;
+
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace OpenMeter.App;
+
+/**
+ * Installed Apps API.
+ */
+@route("/api/v1/apps/{id}/stripe")
+@tag("AppsStripe")
+interface AppStripe {
+  /**
+   * Stripe webhook.
+   */
+  @post
+  @route("/webhook")
+  @operationId("appStripeWebhook")
+  appStripeWebhook(@path id: ULID, @body body: StripeWebhookEvent): StripeWebhookResponse | CommonErrors;
```

Comment:
Does this have to be a public API? Or is it just a webhook we register with Stripe?

Reply:
yes, must be public so Stripe can call it. It handles webhooks.